R/pkg/R/functions.R

```diff
@@ -2875,9 +2875,9 @@ setMethod("ifelse",
 #' @details
 #' \code{cume_dist}: Returns the cumulative distribution of values within a window partition,
 #' i.e. the fraction of rows that are below the current row:
-#' number of values before (and including) x / total number of rows in the partition.
+#' (number of values before and including x) / (total number of rows in the partition)
 #' This is equivalent to the \code{CUME_DIST} function in SQL.
```

Comment:
sure, that should be fine - I mean that it is a equation of some sort something divided by something. I guess that is hard to read in R doc.